impl.go

```diff
@@ -157,7 +163,9 @@ func (p Pkg) fullType(e ast.Expr) string {
 			// the type isn't exported, there's no point trying
 			// to implement it anyway.
 			if n.IsExported() {
```

Comment:
Combine the conditions.

```go
if n.IsExported() && p.Package.Dir != *flagSrcDir {
  ....
}
```

Also, update the comment above to explain why the second half of the condition is present.